src/write_gcm.c

```diff
@@ -3861,7 +3861,11 @@ static int wg_rebase_item(
 
   // If any counter wrapped, everybody resets.
```

Comment:
nit: the comments on this and the preceding block seem flipped. I'd make the one above (3851-2) something like

> Check to see if any counters have decreased from their last value (indicating a wrap or a reset). If one has, we'll reset everyone.

and bring the existing one down to this block.

Reply:
Good suggestion, thanks. Not sure what you meant by "bring the existing one down to this block" — I think the initial comment was actually supposed to cover both blocks, which is what I did. PTAL.

Reply:
love it